go.mod

```diff
@@ -7,12 +7,12 @@ require (
 	github.com/gizak/termui/v3 v3.1.0
 	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/mattn/go-runewidth v0.0.4
-	github.com/nsf/termbox-go v0.0.0-20200418040025-38ba6e5628f1
+	github.com/nsf/termbox-go v0.0.0-20200418040025-38ba6e5628f1 // indirect
 	github.com/shibukawa/configdir v0.0.0-20170330084843-e180dbdc8da0
 	github.com/shirou/gopsutil v2.20.3+incompatible
 	github.com/stretchr/testify v1.4.0
 	github.com/xxxserxxx/opflag v1.0.5
-	golang.org/x/sys v0.0.0-20200316230553-a7d97aace0b0
+	golang.org/x/sys v0.0.0-20200316230553-a7d97aace0b0 // indirect
```

Comment:
This was also pinned to a specific version -- see e72e7be141846948d02da4ec58fe6f1166030e26